ddtrace/contrib/requests/connection.py

```diff
@@ -55,6 +53,15 @@ def _wrap_request(func, instance, args, kwargs):
     url = kwargs.get('url') or args[1]
     headers = kwargs.get('headers', {})
     parsed_uri = parse.urlparse(url)
+    # sanitize url of query
+    sanitized_url = parse.urlunparse((
```

Comment:
:+1:

I was originally thinking just `parsed_uri.path` or something, but this is much better.